docs/development/developer_guide.md

```diff
@@ -47,11 +47,16 @@ a K8s cluster. Set your environment:
 
 ```sh
 export KUBECONFIG=$(echo ~/.kube/config)
-export KUBEFLOW_NAMESPACE=$(your_namespace)
+export KUBEFLOW_NAMESPACE=trainingoperator
 ```
 
 - KUBEFLOW_NAMESPACE is used when deployed on Kubernetes, we use this variable to create other resources (e.g. the resource lock) internal in the same namespace. It is optional, use `default` namespace if not set.
 
+You can create a `kind` cluster by running
+```sh
+kind create cluster --name $KUBEFLOW_NAMESPACE
+```
+
 ### Create the TFJob CRD
 
 After the cluster is up, the TFJob CRD should be created on the cluster.
```

Comment:
Could we remove this section (### Create the TFJob CRD) since it seems that we already do the same below:

```
Deploy the operator with: 
\```sh 
kubectl apply -k ./manifests/overlayes/standalone
\```
```